src/main/java/org/apache/commons/imaging/formats/tiff/TiffImagingParameters.java

```diff
@@ -114,38 +114,98 @@ public void setExif(TiffOutputSet exif) {
         this.exif = exif;
     }
 
-    public Integer getSubImageX() {
-        return subImageX;
+    /**
+     * Sets parameters for performing a partial read operation on an image. This
+     * method is useful for reducing memory and run-time overhead when accessing
+     * large source images.
+     * <p>
+     * Note that the corner x and y coordinates must be positive integers (zero
+     * or greater). The width and height must be greater than zero.
+     *
+     * @param x pixel coordinate of the upper-left corner of the source image,
+     * must be zero or greater.
+     * @param y pixel coordinate of the upper-left corner of the source image,
+     * must be zero or greater.
```

Comment:
Are `x` and `y` both the upper-left corner? I thought maybe `y` would be the lower-right corner?

Reply:
Oh, d'oh! Sorry @gwlucastrig. (x, y) for the top-left corner. I got confused thinking x was a point itself, then y would be another point to create a rectangle. Sorry. Nothing to see here :grimacing: